### PR TITLE
VIITE_1815 Nodes and junctions scripts v.1

### PR DESCRIPTION
**Pull request by elenaols**

### CODE REVIEW

File: digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql [hunk 1]
```diff
@@ -0,0 +1,109 @@
+  CREATE TABLE ROADWAY_POINT
+   (	ID NUMBER NOT NULL ENABLE,
```
Comment (skosonen) [inline digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql]: No need "ENABLE"

File: digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	ADDR_M NUMBER NOT NULL ENABLE,
+	VALID_FROM DATE NOT NULL ENABLE,
+	VALID_TO DATE,
+	CREATED_BY VARCHAR2(32 BYTE) NOT NULL ENABLE,
+	CREATED_TIME TIMESTAMP (6) DEFAULT current_timestamp NOT NULL ENABLE,
+	 CONSTRAINT ROADWAY_POINT_UK1 UNIQUE (ROADWAY_NUMBER, ADDR_M)
+  USING INDEX PCTFREE 10 INITRANS 2 MAXTRANS 255 COMPUTE STATISTICS
+  TABLESPACE USERS  ENABLE,
+	 CONSTRAINT ROADWAY_POINT_PK PRIMARY KEY (ID)
+  USING INDEX PCTFREE 10 INITRANS 2 MAXTRANS 255 COMPUTE STATISTICS
```
Comment (skosonen) [inline digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql]: No need these detailed configurations here.

File: digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql (hunk 1 above)
Comment (elenaols) [reply digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql]: I retrieve ddls straight from sql developer. Should I delete 'enable' and index configs?

Comment (skosonen) [reply digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql]: Yes, I prefer keeping these simple.

Comment (elenaols) [reply digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql]: Done

**Pull request merged**